database/user.go

```diff
@@ -48,6 +48,8 @@ const (
 var (
 	// True is a helper for when we need to pass a *bool to MongoDB.
 	True = true
+	// False is a helper for when we need to pass a *bool to MongoDB.
+	False = false
```

Comment:
why is this needed again? can't you just type `false` whenever you need it in mongo queries?